PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -27,6 +27,17 @@ struct SettingsForm: View {
     var body: some View {
         Form {
             Group {
+                if model.featureFlags.isAssigned(flag: .debugMenu) {
```

Comment:
Perhaps `featureFlags` could be made internal, and the `isAssigned` functionality could be moved into the view model. This would at least eliminate the implementation detail of the feature flags service.